packages/page-accounts/src/modals/Create.tsx

```diff
@@ -134,35 +132,11 @@ function updateAddress (seed: string, derivePath: string, seedType: SeedType, pa
   };
 }
 
-export function downloadAccount ({ json, pair }: CreateResult): void {
-  const blob = new Blob([JSON.stringify(json)], { type: 'application/json; charset=utf-8' });
-
-  FileSaver.saveAs(blob, `${pair.address}.json`);
-}
-
 function createAccount (seed: string, derivePath: string, pairType: PairType, { genesisHash, name, tags = [] }: CreateOptions, password: string, success: string): ActionStatus {
-  // we will fill in all the details below
-  const status = { action: 'create' } as ActionStatus;
-
-  try {
-    const result = keyring.addUri(getSuri(seed, derivePath, pairType), password, { genesisHash, isHardware: false, name, tags }, pairType === 'ed25519-ledger' ? 'ed25519' : pairType);
-    const { address } = result.pair;
-
-    status.account = address;
-    status.status = 'success';
-    status.message = success;
-
-    InputAddress.setLastValue('account', address);
-
-    if (getEnvironment() === 'web') {
-      downloadAccount(result);
-    }
-  } catch (error) {
-    status.status = 'error';
-    status.message = (error as Error).message;
-  }
+  const getResult = () =>
```

Comment:
this actually produces side effects which constitute the main operation of adding an account to the keyring. thus
I'd avoid `get....` and rather go for `commitAccount` or `pushAccountToKeyring` etc. any ideas?

Reply:
Hmm, `commitAccount` sounds good. I will go for it